src/gui/loglistwidget.h

```diff
@@ -31,31 +31,33 @@
 #define LOGLISTWIDGET_H
 
 #include <QListWidget>
+#include "base/logger.h"
 
 QT_BEGIN_NAMESPACE
 class QKeyEvent;
 QT_END_NAMESPACE
 
-class LogListWidget : public QListWidget
+class LogListWidget: public QListWidget
 {
     Q_OBJECT
 
 public:
-  explicit LogListWidget(int max_lines = 100, QWidget *parent = 0);
+    // -1 is the portable way to have all the bits set
+    explicit LogListWidget(int maxLines, const Log::MsgTypes &types = Log::MsgTypes(-1), QWidget *parent = 0);
```

Comment:
Oops. That was before I added the Log::ALL.
PS: I should remove the `explicit` keyowrd too, right?